sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/auth/AuthenticationIT.java

```diff
@@ -135,7 +135,7 @@ void shouldSendDataForAuthorizationToOpa() {
                         "$.input.headers.authorization[0]",
                         equalTo("Bearer " + authentication.token())))
                 .withRequestBody(
-                    matchingJsonPath("$.input.headers.connection[0]", equalTo("Keep-Alive"))));
+                    matchingJsonPath("$.input.headers.connection[0]", equalTo("keep-alive"))));
```

Comment:
NIT: Imho technically correct and more sustainable would be to ignore the case instead of adjusting the expectation:

```suggestion
                    matchingJsonPath("$.input.headers.connection[0]", equalToIgnoreCase("Keep-Alive"))));
```